cloud/integrations.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Integrate Timescale Cloud services with third-party monitoring tools
+excerpt: Export telemetry metrics to DataDog or AWS CloudWatch
+product: cloud
+keywords: [integrations, DataDog, AWS CloudWatch]
+---
+
+import CloudIntegrationsExporterRegion from 'versionContent/_partials/_cloud-integrations-exporter-region.mdx';
+
+# Integrate Timescale Cloud services with third-party monitoring tools
+
+You can export your service telemetry to a third-party monitoring tool, such as
+[DataDog][datadog] or [AWS CloudWatch][cloudwatch]. To do so, create a data
+exporter and attach your service to it.
+
+## Create a data exporter
+
+Create a data exporter to send telemetry data to your preferred provider.
+
+<procedure>
+
+### Creating a data exporter to DataDog
+
+<CloudIntegrationsExporterRegion />
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `Datadog`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Add a DataDog API key. If you don't have an API key yet, click `Click here
+    to generate a DataDog API key`.
+1.  Under Site, choose your DataDog region. You can choose a region to meet any
+    regulatory requirements or application needs you might have.
+1.  Click `Create endpoint`.
+
+<!-- FIXME: insert screenshot here -->
+
+</procedure>
+
+<procedure>
+
+### Creating a data exporter to AWS CloudWatch
+
+<CloudIntegrationsExporterRegion />
+
+1.  In the Timescale Cloud console, navigate to `Integrations`.
+1.  Click `Create exporter`.
+1.  Under `Choose a provder`, choose `AWS CloudWatch`.
+1.  Choose an AWS region for your exporter to live within Timescale Cloud. The
+    exporter is only available to database services in the same AWS region.
+1.  Name your exporter. This name appears in the Cloud console, so choose a
+    descriptive name.
+1.  Define names for your CloudWatch log group, log stream, and namespace. If
+    you're uncertain, use the default values. For more information on naming log
+    groups and logs streams, see [the AWS CloudWatch
+    docs][cloudwatch-log-naming].
+1.  Enter your AWS credentials. To get your AWS keys, see [Getting access keys
+    for AWS CloudWatch][aws-access-keys].
+1.  Select an AWS Region for your CloudWatch instance.
+1.  **(optional)** Define an IAM role to upload segments to a
+    different account.
```

Comment:
@lukas-bernert Can you clarify what "upload segments to a different account" means here?

Reply:
If you have two AWS accounts (e.g. one for production and one for development) it makes sense to have users for each developer in the dev account. However, you might not want to give every developer a user for the production account. Instead you can create an IAM role for production with restricted access. Through this role developers can now access the production account.

Better explanation can be found here: https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html